estate/models/estate_property.py

```diff
@@ -0,0 +1,86 @@
+from odoo import api, fields, models
+from dateutil.relativedelta import relativedelta
+
+from odoo.exceptions import UserError, ValidationError
+from odoo.tools import float_compare
+from .helper import format_selection
+
+from odoo.fields import Many2many, One2many
+
+
+class EstateProperty(models.Model):
+    _name = 'estate.property'
+    _description = 'Estate Property modelisation'
+    _order = 'id desc'
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(default=lambda x: fields.Date.today() + relativedelta(months=3), copy=False)
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True, copy=False)
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection(string='Orientation',
+                                          selection=format_selection(['north', 'south', 'east', 'west']))
```

Comment:
Funny, does it work alright like that ? First time I see someone do that aha. Still it's better to have the selection as a list of tuple (key, value) as stated in the doc.
```suggestion
    garden_orientation = fields.Selection(
        string='Orientation',
        selection=format_selection([
            ('north', "North"), 
            ('south', "South"), 
            ('east', "East"),
            ('west', "West"),
        ]),
    )
```